arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -0,0 +1,14114 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-08-10",
+    "title": "SiteRecoveryManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems": {
+      "get": {
+        "tags": [
+          "ReplicationProtectedItems"
+        ],
+        "summary": "Gets the list of Replication protected items.",
+        "description": "Gets the list of ASR replication protected items in the protection container.",
+        "operationId": "ReplicationProtectedItems_ListByReplicationProtectionContainers",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
```

Comment:
Given the kind of usage we have for parameters like fabricName, containerName they change in general for every second operation our customers can call, to minimize confusion we have kept all parameter which would remain constant for a instance of initialized client in global list and others as local. This is also consistent with the current usage.